components/mpas-ocean/src/analysis_members/mpas_ocn_conservation_check.F

```diff
@@ -699,6 +715,9 @@ subroutine energy_conservation(domain, err)
               + accumulatedRiverRunoffTemperatureFlux   *rho_sw*cp_sw &
               + accumulatedIcebergTemperatureFlux*rho_sw*cp_sw
               ! note, accumulatedLandIceFrazilHeatFlux not added because already in accumulatedFrazilHeatFlux
+         if (trim(config_subglacial_runoff_mode) == 'data') then
+            netEnergyFlux = netEnergyFlux + accumulatedSubglacialRunoffTemperatureFlux * rho_sw*cp_sw
+         end if
```

Comment:
I think we can treat this in the same way as landIceHeatFlux -- remove the if-clause here because it will be 0 if there are no SGR fluxes.

Reply:
The if clause is necessary because accumulatedSubglacialRunoffTemperatureFlux is defined in a package dataSubglacialRunoffFluxPKG only if trim(config_subglacial_runoff_mode) == 'data'